packages/vue-app/template/router.js

```diff
@@ -86,7 +86,18 @@ Vue.use(Router)
 const scrollBehavior = <%= serializeFunction(router.scrollBehavior) %>
 <% } else { %>
 if (process.client) {
-  window.history.scrollRestoration = 'manual'
+  if ('scrollRestoration' in window.history) {
+    window.history.scrollRestoration = 'manual'
+    // reset scrollRestoration to auto when leaving page, allowing page reload and back-navigation from other pages
```

Comment:
Thanks @ahus1 -- this looks good to me.

Please trim the comments to about 80 columns, for better readability.

Reply:
@galvez - comments are now re-formatted. Tests are still green. Changes force-pushed.